### ISSUE TITLE
"cluster: no services found" during buildpack test

### ISSUE THREAD

**Issue by titanous** [flynn/flynn#906]
```text
++ 23:16:45.931 /usr/bin/git push flynn master
remote: [1G[K-----> Building nodejs-flynn-example...        
remote: Build failed: cluster: no servers found        
To ssh://git@flynn-54d08a68f3a16a82.local:2222/nodejs-flynn-example.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'ssh://git@flynn-54d08a68f3a16a82.local:2222/nodejs-flynn-example.git'
```

https://s3.amazonaws.com/flynn-ci-logs/20150130231514-13bd6d58-build-376e150ab4de7365232b447e3333080047a3d17c-2015-01-30-23-20-13.txt

Timestamp corresponds with connection errors in the etcd logs.

**Comment by lmars** [comment]
The underlying etcd connection errors which cause these failures look like:

```
2015/03/12 21:24:08 sender: error posting to 267731ccfa5f94fd: EOF
2015/03/12 21:24:08 sender: the connection with 267731ccfa5f94fd became inactive
2015/03/12 21:24:08 sender: the connection with 267731ccfa5f94fd became active
2015/03/12 21:24:14 etcdhttp: unexpected error: etcdserver: request timed out
```

See https://s3.amazonaws.com/flynn-ci-logs/20150312211728-6a18daaf-build-348b59ea0719ff3369f93151f38175817cf82a79-2015-03-12-21-30-26.txt

**Comment by titanous** [comment]
Closing, as this appears to have been fixed by #1254.
